src/mongo/db/index/index_descriptor.cpp

```diff
@@ -171,8 +171,41 @@ IndexDescriptor::Comparison IndexDescriptor::compareIndexOptions(
     const IndexCatalogEntry* existingIndex) const {
     auto existingIndexDesc = existingIndex->descriptor();
 
+    //btree index check, avoid create duplicate btree indexes, Avoid affecting performance by the deplicate index.
```

Comment:
avoid create -> avoid creating.

"Avoid affecting performance by the deplicate index." is not necessary as it's obvious and performance is not the only side effect.